src/Components/NavBar/NavBar.jsx

```diff
@@ -35,14 +41,16 @@ function NavBar() {
             <i className="fas fa-user icon"></i>
           </Link>
 
-          <div className="badge" onClick={()=> navigate('/cart')}>
-              {token && <i className="fas fa-shopping-cart icon icons-badge"></i>}
-              {token && <span className=" badge-icon-number">{cart?.length}</span>}
+          <div className="badge" onClick={() => navigate("/cart")}>
+            {token && <i className="fas fa-shopping-cart icon icons-badge"></i>}
+            {token && (
+              <span className=" badge-icon-number">{cart?.length}</span>
+            )}
           </div>
 
-          <div className="badge">
-              {token && <i className="far fa-heart ico icon icons-badge"></i>}
-              {token && <span className=" badge-icon-number">0</span>}
+          <div className="badge" onClick={() => navigate('/wishlist')}>
```

Comment:
Use a react-router `Link` instead of navigate element.
Why? 
A wishlist is a separate page and for linking to separate pages we should use `a` tag. This is good accessibility practice. 

Also in the official react router docs it was mentioned that you should have a good reason to use navigate. 
Here you can use a link.

Reply:
> Use a react-router `Link` instead of navigate element. Why? A wishlist is a separate page and for linking to separate pages we should use `a` tag. This is good accessibility practice.
> 
> Also in the official react router docs it was mentioned that you should have a good reason to use navigate. Here you can use a link.

ok sure will do that